### PR TITLE
[MRG+1] Add `fittedvalues` to public ARIMA api

### PR DESCRIPTION
**Pull request by tgsmith61591**
# Description

Expose `fittedvalues` in the public `ARIMA` API

Fixes #(issue)

#493 

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation change

### CODE REVIEW

File: doc/whats_new.rst [hunk 1]
```diff
@@ -7,6 +7,17 @@ What's new in pmdarima
...
+* Remove deprecation warnings for the ``exogenous`` keyword introduced in 1.8.0
+
+* Remove deprecation warnings for the ``sarimax_kwargs`` keyword introduced in 1.5.1
+
+* Bump numpy dependency to >= 1.21
+
+* Expose ``fittedvalues`` in the public API. See `#493 <https://github.com/alkaline-ml/pmdarima/issues/493>`_
+
 `v1.8.6 <http://alkaline-ml.com/pmdarima/1.8.6>`_
 -------------------------------------------------
```
Comment (aaronreidsmith) [inline doc/whats_new.rst:23]: If we are skipping 1.8.6, can you move the aarch64 note to 2.0.0?

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 joblib>=0.11
 Cython>=0.29,!=0.29.18
-numpy>=1.19.3
+numpy>=1.21
```
Comment (aaronreidsmith) [inline requirements.txt:3]: Do we also need to update `build_tools/build_requirements.txt`?

**Review by aaronreidsmith — APPROVED** [review]
2 small things. Otherwise looks good

File: requirements.txt (hunk 2 above)
Comment (tgsmith61591) [reply requirements.txt:3]: Good catch. I don't remember why we even have those separated tbh

File: doc/whats_new.rst (hunk 1 above)
Comment (tgsmith61591) [reply doc/whats_new.rst:23]: Oh snap, thought we had released that 😬

**Pull request merged**